addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1435,10 +1524,30 @@ MockServer.include({
                 'res.partner',
                 [['id', 'in', message.partner_ids]],
             );
+            const message_reactions = this._getRecords('mail.message.reaction', [
+                ['message_id', '=', message.id],
+            ]);
+            const messageReactionGroupsRaw = new Map();
+            for (const reaction of message_reactions) {
+                let messageReactionGroup = messageReactionGroupsRaw.get(reaction.content);
+                if (!messageReactionGroup) {
+                    messageReactionGroup = {
+                        'messageId': reaction.message_id,
+                        'content': reaction.content,
+                        'count': 0,
+                        'partners': [['insert-and-replace', [{'id': reaction.partner_id, 'name': 'Test Demo'}]]],
+                    };
+                    messageReactionGroupsRaw.set(reaction.content, messageReactionGroup);
+                    messageReactionGroup = messageReactionGroupsRaw.get(reaction.content);
+                }
+                messageReactionGroup['count'] += 1;
```

Comment:
missing handling of guests

Reply:
Same remark about test env for guests (waiting for @seb-odoo to confirm)